CONTRIBUTORS_GUIDE.md

```diff
@@ -0,0 +1,38 @@
+# Contributor's Guide
+written by @MaxReinisch
+
+
+## Getting Started
```

Comment:
Getting started doing what? Installing the dev environment? Please try to be more specific
You could also include the command to run the unit tests. This also confirms that all are setup correctly up to point.


You could have another getting started guide to load the extension in your browser (I know that its relatively trivial but still, it could be useful for some people).